packages/flutter/test/material/app_bar_theme_test.dart

```diff
@@ -64,18 +78,24 @@ void main() {
         brightness: brightness,
         elevation: elevation,
         iconTheme: iconThemeData,
-        textTheme: textTheme,)
-      ),
+        actionsIconTheme: actionsIconThemeData,
+        textTheme: textTheme,
+        actions: <Widget>[
+          IconButton(icon: const Icon(Icons.share), onPressed: () {})
```

Comment:
What about the case where you manually set the icon's color?

Reply:
Good call, added a separate test for that case